### PR TITLE
Adds buy ticket button on website

### LINKED ISSUE #76: Add a button to buy tickets for conference

### PR DESCRIPTION
**Pull request by CuriousLearner**
Resolves #76

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -125,6 +125,9 @@
 										<span>Delhi-NCR</span>
 									</li>
 								</ul>
+								<div class="tg-btnarea">
+									<a class="tg-btn tg-btnbookseat" style="color: #FFF;" href="https://in.explara.com/e/pycon-india-2017" target="_blank">Buy Tickets</a>
```
Comment (sayanchowdhury) [inline index.html]: Can you replace `target=_blank" with `rel="noopener noreferrer`?

File: templates/index.html [hunk 2]
```diff
@@ -39,6 +39,9 @@
 										<span>Delhi-NCR</span>
 									</li>
 								</ul>
+								<div class="tg-btnarea">
+									<a class="tg-btn tg-btnbookseat" style="color: #FFF;" href="https://in.explara.com/e/pycon-india-2017" target="_blank">Buy Tickets</a>
```
Comment (sayanchowdhury) [inline templates/index.html]: Same here also.

**Review by sayanchowdhury — CHANGES_REQUESTED** [review]

**Comment by CuriousLearner** [comment]
@sayanchowdhury Please review again.

**Comment by rajataaron** [comment]
Looks good to me ready to merge . 
What you say @sayanchowdhury

**Comment by SaptakS** [comment]
@CuriousLearner can you squash the commits? Apart from that looks good to me.

**Comment by CuriousLearner** [comment]
@SaptakS Yes, we'll be squashing commits. One PR == One Commit! :)

**Comment by CuriousLearner** [comment]
I'm merging this since we need to deploy this change.

@sayanchowdhury if you feel this needs to be changed, please let me know.

**Review by sayanchowdhury — COMMENTED** [review]
:fireworks:

**Pull request merged**